dpcpp/components/prefix_sum_kernels.dp.cpp

```diff
@@ -50,18 +51,15 @@ namespace dpcpp {
 namespace components {
 
 
-using BlockCfg = ConfigSet<11>;
+constexpr auto block_cfg_list = block_cfg_list_t();
```

Comment:
no need to emit a symbol for this (also might cause ODR issues if we have different values for the same variable in different files)
```suggestion
static constexpr auto block_cfg_list = block_cfg_list_t();
```